### ISSUE TITLE
Advanced Models don't work at all for Keyword Recognition inside the Java SDK

### ISSUE THREAD

**Issue by JangoCG** [Azure-Samples/cognitive-services-speech-sdk#2590]
**Describe the bug**
When I load an advanced model in the Java SDK the Keyword Recogntion does not work at all, the keyword is **never** recognized It will however work perfectly fine with a basic model. The code for this is Java Sample provided in your sample repo.

**To Reproduce**
1. Checkout Java  Embedded Sample
2. Replace the provided model with default model of your choice
3. Start the the Sample and type in `3` in the cli to start keyword recogntion
4. it will work with an basic model
5. if you replace the basic model with an advanced model it will not work, the keyword is **never recognized**

I tried it with different advanced models. None of them work. However those advanced models work perfectly fine in the speech studio on the `test model` tab. So I don't think there is anything wrong with the models

**Expected behavior**

It should work with advanced models

**Version of the Cognitive Services Speech SDK**

Which version of the SDK are you using.

```xml
   <groupId>com.microsoft.cognitiveservices.speech</groupId>
      <artifactId>client-sdk-embedded</artifactId>
      <version>1.40.0</version>
```

**Platform, Operating System, and Programming Language**

 - OS: MAC
 - Hardware: ARM  (M2 Macbook Pro)
 - Programming language: Java

**Comment by JangoCG** [comment]
After playing arround with the SDK and using the Keyword Recognzier instead of the Speech Recognizer I was able to reproduce:
https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/2571

So I am pretty sure with the speech recognizer the underlying issue is the same and the java SDK is also affected

**Comment by pankopon** [comment]
Hi, these issues are due to the format of (advanced) keyword models changed on Speech Studio while the Speech SDK support for it was not completed. We'll fix it in the 1.41.0 release due next month.

In this case the necessary library file is in the Java SDK package but not loaded. Until the release with a fix is available, please use the following workaround:
* extract `ASSETS/osx-arm64/libMicrosoft.CognitiveServices.Speech.extension.kws.ort.dylib` from `client-sdk-1.40.0.jar` (it's a zip compressed archive so use `unzip` or similar)
  * if you have built the latest [embedded speech sample](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/java/jre/embedded-speech) with Maven, the installed jar archive should be found as ` $HOME/.m2/repository/com/microsoft/cognitiveservices/speech/client-sdk-embedded/1.40.0/client-sdk-embedded-1.40.0.jar`
  * alternatively you can download the original jar from https://repo1.maven.org/maven2/com/microsoft/cognitiveservices/speech/client-sdk/1.40.0/
* copy `libMicrosoft.CognitiveServices.Speech.extension.kws.ort.dylib` to some directory in `java.library.path` (e.g. the current working directory `.` where you run the sample)
  * you can see the current paths with `java -XshowSettings:properties`
* then run the sample on the command-line with `java -jar ./target/EmbeddedSpeechSamples-snapshot-jar-with-dependencies.jar` and keyword recognition using advanced models will work

**Comment by pankopon** [comment]
Fixed in the Speech SDK 1.41.1 release.
